share/spack/gitlab/cloud_pipelines/stacks/ml-linux-x86_64-rocm/spack.yaml

```diff
@@ -0,0 +1,91 @@
+spack:
+  view: false
+  packages:
+    all:
+      require:
+      - target=x86_64_v3
+      - ~cuda
+      - +rocm
+      - amdgpu_target=gfx90a
+    gl:
+      require: "osmesa"
+    mpi:
+      require: openmpi
+
+  specs:
+    # Horovod
+    - py-horovod
```

Comment:
Might be building with torch, you can tell it to use TF

Reply:
I commented it out since it requires py-tensorflow-estimator if it uses TF. That wouldn't work since py-tensorflow-estimator doesn't have a 2.16 version